kitsune/search/v2/management/commands/es7_reindex.py

```diff
@@ -32,6 +43,9 @@ def handle(self, *args, **kwargs):
 
         progress_msg = "Indexed {progress} out of {count}"
 
+        if kwargs["print_sql_count"]:
```

Comment:
nope :)

Reply:
I was planning on adding some documentation saying something along the lines of "after you write/edit an ES document, run an index with `--print-sql-count` to ensure you're not throwing hundreds of requests at the DB. If you are, consider adding a `select_` or `prefetch_related` to the document's get_queryset method to avoid that."

`django.db.connection.queries` needs `DEBUG=True` to do anything anyway, so being behind an argument as well, I'm not sure this could do anything negative on prod.

Reply:
I am mostly referent to the conditional in order to do the import